App/Redux/TextileRedux.js

```diff
@@ -54,9 +60,18 @@ export const onboardedSuccess = state => {
   return state.merge({ onboarded: true })
 }
 
-export const handleImageAdded = (state, {thread, hash, remotePayloadPath}) => {
+// Used to ignore certain URIs in the CameraRoll
+export const handleImageIgnore = (state, {uri}) => {
+  const processed = state.camera && state.camera.processed ? state.camera.processed : []
+  const camera = {processed: [uri, ...processed]}
```

Comment:
Would be good to expand in the rest of the `camera` object in case we add anything to it in the future. If we leave this as as, any additional keys will be deleted. So something like:
```
const camera = {...state.camera, processed: [uri, ...processed]}
```

Reply:
cool